p2-profile/pom.xml

```diff
@@ -200,6 +200,9 @@
                                 <featureArtifactDef>
                                     org.wso2.ei:org.wso2.micro.integrator.identity.xacml.mediator.feature:${project.version}
                                 </featureArtifactDef>
+                                <featureArtifactDef>
+                                    org.wso2.ei:org.wso2.micro.integrator.mediator.publishevent.server.feature:${project.version}
```

Comment:
Please clean up the existing publishevent feature .

Reply:
Done in bd10bd3